README.md

```diff
@@ -141,12 +140,12 @@ When the parameters are absent in the ROS parameter server, default values are u
 #### Published Topics
 | Topic Name | Message Type | Description |
 | ---------- | ------------ | ----------- |
-| *Configurable* (default="video/encoded") | kinesis_video_msgs/KinesisVideoFrame | The node will publish to a topic of a given name. Each message being published contains a chunk of the video stream, usually per video frame. |
+| *Configurable* (default="video/encoded") | kinesis_video_msgs/msg/KinesisVideoFrame | The node will publish to a topic of a given name. Each message being published contains a chunk of the video stream, usually per video frame. |
```

Comment:
Is having a /msg/ subfolder a ROS2 thing? If not I think we should fix the msgs package to not have this to be consistent with the base message types, because it will confuse a lot of users having to add that extra `/msg` in there just for these messages when they usually don't have to.

Reply:
This is how the messages are represented in header files (e.g. import <kinesis_video_msgs/msg/KinesisVideoFrame.h>`, however from the command line you would do `ros2 topic pub <topic> kinesis_video_msgs/KinesisVideoFrame`, so maybe we should just stick to that.